nevergrad/benchmark/experiments.py

```diff
@@ -63,22 +64,23 @@ def yawidebbob(seed: Optional[int] = None) -> Iterator[Experiment]:
     for optim in optims:
         for function in functions:
             for budget in [50, 500, 5000, 50000]:
-                    for nw in [1, 100]:
-                        xp = Experiment(function, optim, num_workers=nw,
-                                        budget=budget, seed=next(seedg))
-                        if not xp.is_incoherent:
-                            yield xp
+                for nw in [1, 100]:
+                    xp = Experiment(function, optim, num_workers=nw,
+                                    budget=budget, seed=next(seedg))
+                    if not xp.is_incoherent:
+                        yield xp
     # Discrete, unordered.
     for nv in [10, 50, 200]:
-        for optim in optims:
-            for nw in [1, 10]:
-                for budget in [500, 5000]:
-                    for discrete_func in [corefuncs.onemax, corefuncs.leadingones, corefuncs.jump]:
-                        for arity in [2, 7]:
-                            variables = list(ng.p.TransitionChoice(list(range(arity))) for _ in range(nv))
-                            instrum = ng.p.Instrumentation(*variables)
-                            yield Experiment(ExperimentFunction(discrete_func, instrum), optim, num_workers=nw, budget=budget, seed=next(seedg))
-    mofuncs: List[PackedFunctions] = []
+        for arity in [2, 7]:
+            variables = list(ng.p.TransitionChoice(list(range(arity))) for _ in range(nv))
+            instrum = ng.p.Instrumentation(*variables)
```

Comment:
The only important stuff is here: creating an intrumentation can be time consuming because it creates data, if we create it asap we save 3 * 2 * 2 * num_optims creations, which saves 1 min of computation (out of 1min)

Reply:
Excellent :-)